device-types/LANCOM/1793VA-4G.yaml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: LANCOM
+model: 1793VA-4G
+slug: lancom-1793va-4g
+part_number: '62116'
+u_height: 1
+is_full_depth: false
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
Hi,
I added the weight.

For the airflow i only found the hint "1 silent fan". I didn't found any directions. From the pictures I assume it is right to left oder left to right. But that's just a guess.
Unfortunately, I don't have physical access to the device either.
How should we proceed here?